cmd/export.go

```diff
@@ -21,13 +22,19 @@ var channelWithChildren []string
 var outputDir string
 var metadataOnly bool
 var startingDate string
+var includeImages bool
+var includeContainers bool
+var orgidOnly uint
 
 func init() {
 	exportCmd.Flags().StringSliceVar(&channels, "channels", nil, "Channels to be exported")
 	exportCmd.Flags().StringSliceVar(&channelWithChildren, "channel-with-children", nil, "Channels to be exported")
 	exportCmd.Flags().StringVar(&outputDir, "outputDir", ".", "Location for generated data")
 	exportCmd.Flags().BoolVar(&metadataOnly, "metadataOnly", false, "export only metadata")
 	exportCmd.Flags().StringVar(&startingDate, "packagesOnlyAfter", "", "Only export packages added or modified after the specified date (date format can be 'YYYY-MM-DD' or 'YYYY-MM-DD hh:mm:ss')")
+	exportCmd.Flags().BoolVar(&includeImages, "images", false, "Export OS images and associated metadata")
+	exportCmd.Flags().BoolVar(&includeContainers, "containers", false, "Export containers metadata")
+	exportCmd.Flags().UintVar(&orgidOnly, "orgId", 0, "Export only for organization id")
```

Comment:
I think we should receive the org label instead of org id. And should we receive a list of Org Labels?